react/src/pages/ServingListPage.tsx

```diff
@@ -59,9 +58,8 @@ const ServingListPage: React.FC<PropsWithChildren> = ({ children }) => {
   const curProject = useCurrentProjectValue();
   const [isOpenServiceLauncher, setIsOpenServiceLauncher] = useState(false);
   const [isOpenColumnsSetting, setIsOpenColumnsSetting] = useState(false);
-  const [selectedModelService, setSelectedModelService] = useState<Endpoint>();
-  const [isOpenModelServiceSettingModal, setIsOpenModelServiceSettingModal] =
-    useState(false);
+  const [selectedModelService, setSelectedModelService] =
+    useState<Endpoint | null>(null);
 
```

Comment:
Let's seperate the `selectedModelService` into `editingModelService` and `deletingModelService`.

It also can be used to determine wether modal is open or not.

For example,
```
 <BAIModal
        open={!deletingModelService}
        title={t('dialog.title.LetsDouble-Check')}
```

Reply:
Hmm, I think It's already set according to row in table data, and I think deletingModelService is already set as a name of "`isOpenModelServiceTerminatingModal`"